web/jsonb/jsonbbasics/src/main/java/javaeetutorial/jsonbmodel/JsonbBean.java

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
+ *
+ * You may not modify, use, reproduce, or distribute this software except in
+ * compliance with  the terms of the License at:
+ * https://github.com/javaee/tutorial-examples/LICENSE.txt
+ */
+package javaeetutorial.jsonbmodel;
+
+import javaeetutorial.jsonbmodel.person.Person;
+import javaeetutorial.jsonbmodel.person.PhoneNumber;
+
+import java.io.*;
+import java.util.*;
+import javax.enterprise.context.SessionScoped;
+import javax.inject.Named;
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+import javax.json.bind.JsonbConfig;
+
+/* This class manages the data from a JSF form, creates 
```

Comment:
I see some inconsistency in comments format. Other files are using different style of comments. Please make comments in this file the same as in others. This is a sample and it should look perfect from the formatting perspective.